README.md

```diff
@@ -30,7 +32,7 @@ This Juju bundle deploys Tempo and a small object storage server, consisting of
 
 - [Tempo Coordinator](https://charmhub.io/tempo-coordinator-k8s) ([source](https://github.com/canonical/tempo-coordinator-k8s-operator))
 - [Tempo Worker](https://charmhub.io/tempo-worker-k8s) ([source](https://github.com/canonical/tempo-worker-k8s-operator))
-- [s3integrator](https://charmhub.io/s3-integrator) ([source](https://github.com/canonical/s3-integrator))
+- [S3 Integrator](https://charmhub.io/s3-integrator) ([source](https://github.com/canonical/s3-integrator))
```

Comment:
Might also be worth noting in the `README` that the bundle won't work without configuring s3-integrator to point at something s3-compatible, be it s3 credentials or another charm that does the s3 work like minio.